tvshows/src/app/shows/page.tsx

```diff
@@ -0,0 +1,15 @@
+'use client'
+import { ShowsList } from '@/components/shared/ShowsList/ShowsList';
+import { getShowLists } from '@/fetchers/show';
+import { Box } from '@chakra-ui/react';
+
+const AllShows = () => {
+  
+  return (
+    <Box flex="1" p="4" display="flex" justifyContent="center">
+    <ShowsList route='/api/shows' getter={getShowLists} />
```

Comment:
Ideally, you wouldn't expose the specifics like 'route' and 'getter' to the outside. You would encapsulate that behaviour inside the component that is responsible for it and proved an interface that just says `top-rated is true or false`.

Another point is that the naming of the component implies this is a presentational component, while it is fetching data. Just makes it a bit weird.